lib/wine_bouncer/extension.rb

```diff
@@ -0,0 +1,20 @@
+module WineBouncer
+  module Extension
+    def oauth2(*scopes)
+      scopes = Array(scopes)
+      description = if respond_to?(:route_seting) # >= grape-0.10.0 
+        route_setting(:description)
+      else
+        @last_description ||= {}
+      end
+      # case WineBouncer.configuration.auth_strategy
```

Comment:
I can't write spec with this branching because `configuration` is required before calling `oauth2` method.